scripts/install

```diff
@@ -21,16 +26,17 @@ ynh_abort_if_errors
 
 domain=$YNH_APP_ARG_DOMAIN
 path_url=$YNH_APP_ARG_PATH
-admin=$YNH_APP_ARG_ADMIN
 is_public=$YNH_APP_ARG_IS_PUBLIC
+admin=$YNH_APP_ARG_ADMIN
 email=$(ynh_user_get_info --username=$admin --key=mail)
+phpversion=$YNH_PHP_VERSION
```

Comment:
Why adding that ?

Reply:
Because I had issues without it. (the problem could come from PHP helper)

Reply:
I thunk the solution is just to add `phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)` after `ynh_add_fpm_config` or having this helper export `phpversion`